bundles/k3s-rhel8/Dockerfile

```diff
@@ -0,0 +1,20 @@
+# THIS ONLY DOWNLOADS THE SELINUX POLICY, not the k3s cli
+FROM centos:8
+
+RUN yum install -y gettext # envsubst
+RUN yum install -y yum-utils # yumdownloader
+
+ADD ./rancher-k3s-common.repo.tmpl /tmp/
+
+ENV rpm_site=rpm.rancher.io
+ENV k3s_rpm_channel=stable
+ENV maj_ver=8
+
+ARG K3S_VERSION
+
+RUN mkdir -p /packages/archives
```

Comment:
how come this isnt in rhel7?

Reply:
It is, [just in a slightly different order](https://github.com/replicatedhq/kURL/blob/feat/exp-k3s-support/bundles/k3s-rhel7/Dockerfile#L6), which is an artifact of the copy-paste from RKE2. I can make it uniform though.